Services/NewsService/Validators/CurrencyRequestValidator.cs

```diff
@@ -0,0 +1,24 @@
+﻿using FluentValidation;
+
+using DTO.NewsRequests.Currency;
+
+namespace NewsService.Validators
+{
+    public class CurrencyRequestValidator : AbstractValidator<CurrencyRequest>
+    {
+        public CurrencyRequestValidator()
+        {
+            RuleFor(r => r)
+                .NotNull()
+                .WithMessage("Request parameters must be not null.");
+
+            RuleFor(r => r.CurrencyCodes)
+                .NotNull()
+                .WithMessage("List with currency codes must be not null.");
+
+            RuleFor(r => r)
```

Comment:
Can't we add multiple rules in one `RuleFor` block? Or you think that this check would cause bugs if `CurrencyCodes `is null?

Reply:
I don't know how is correct